README.md

```diff
@@ -246,6 +246,35 @@ deploy_script:
             -ResourceModuleName $moduleName
 ```
 
+## CodeCoverage reporting with CodeCov.io
+
+### Repos using `-Type 'Default'` for `Invoke-AppveyorTestScriptTask`
+
+1. On the call to `Invoke-AppveyorTestScriptTask`, make sure you have
```

Comment:
Add a note that this is code coverage through Pester and not code coverage with CodeCovIO

Also please add a note about what the difference between running these two is or why you would or would not want to run one or the other.